app/_src/introduction/install-kuma.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Install Kuma
+content_type: how-to
+---
+
+{% tip %}
+This guide is mostly useful for Universal setup, as for Kubernetes we recommend using `kubectl` for managing [resources](/docs/{{ page.version }}/introduction/concepts#resource).
+More in [Kubernetes quickstart guide](/docs/{{ page.version }}/quickstart/kubernetes-demo/).
+{% endtip %}
+
+This is simple guide on how to install {{site.mesh_product_name}} on your machine.
+
+1. Go to the [{{site.mesh_product_name}} packages](https://cloudsmith.io/~kong/repos/{{site.mesh_product_name_path}}-binaries-release/packages/?q=version%3A{{ page.version_data.version }}) 
```

Comment:
`dev` does not work: 

![image](https://github.com/user-attachments/assets/0e6f737b-5413-49c4-95e1-a321941ec853)

for dev you need

https://cloudsmith.io/~kong/repos/kuma-binaries-preview/packages/?sort=-date

Reply:
this might work, but not tested:

```suggestion
{% if_version eq:preview %}
1. Go to the [{{site.mesh_product_name}} packages](https://cloudsmith.io/~kong/repos/{{site.mesh_product_name_path}}-binaries-preview/packages/?sort=-date) 
{% endif_version %}

{% if_version neq:preview %}
1. Go to the [{{site.mesh_product_name}} packages](https://cloudsmith.io/~kong/repos/{{site.mesh_product_name_path}}-binaries-release/packages/?q=version%3A{{ page.version_data.version }}) 
{% endif_version %}
```

Reply:
unfortunately `if_version` works only with specific version if fails on both `preview` and `dev` :(